### PR TITLE
Feat: add http status and code from http cmd run

### LINKED ISSUE #3157: [Feature] return http response status in pkg/builtin/http/http.go HTTPCmd.Run
**Describe the solution you'd like**

希望利用 pkg/stdlib/op.cue中 的 #HTTPGet #HTTPPost 等http操作符，实现 业务服务的健康检查 的workflowStep Definition.

发现在 [./pkg/builtin/http/http.go#L130](https://github.com/oam-dev/kubevela/blob/2da60419fcf84b62149c61018cb91ba469520a63/pkg/builtin/http/http.go?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L130) 的实现中，返回的 response 数据中缺少状态/状态码，对于健康检查的判断，在cue模版中不是很方便。 

建议在返回值添加 status / status_code 的信息。

```golang
// Run exec the actual http logic, and res represent the result of http task
func (c *HTTPCmd) Run(meta *registry.Meta) (res interface{}, err error) {
...
	// parse response body and headers
	return map[string]interface{}{   // #L130
		"body":        string(b),
		"header":      resp.Header,
		"trailer":     resp.Trailer,
		"status":      resp.Status,
		"status_code": string(resp.StatusCode),
	}, err
}
```

### PR DESCRIPTION
**Pull request by ArenaSu**
Signed-off-by: suxiang <704427617@qq.com>

### Description of your changes

Fixes #3157

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: pkg/builtin/http/http.go [hunk 1]
```diff
@@ -128,9 +128,11 @@ func (c *HTTPCmd) Run(meta *registry.Meta) (res interface{}, err error) {
...
 	// parse response body and headers
 	return map[string]interface{}{
-		"body":    string(b),
-		"header":  resp.Header,
-		"trailer": resp.Trailer,
+		"body":        string(b),
+		"header":      resp.Header,
+		"trailer":     resp.Trailer,
+		"status":      resp.Status,
+		"status_code": resp.StatusCode,
```
Comment (wonderflow) [inline pkg/builtin/http/http.go:135]:
```suggestion
		"statusCode": resp.StatusCode,
```

Comment (wonderflow) [reply pkg/builtin/http/http.go:135]: status seems not necessary

Comment (ArenaSu) [reply pkg/builtin/http/http.go:135]: Does status contain error reason messages when error occurred ?

Comment (wonderflow) [reply pkg/builtin/http/http.go:135]: no, it won't. The useful information contained in the response body.

Comment (ArenaSu) [reply pkg/builtin/http/http.go:135]: ok, i'll fix it.

File: pkg/workflow/providers/http/do_test.go [hunk 2]
```diff
@@ -57,6 +57,7 @@ func TestHttpDo(t *testing.T) {
 			body: string
 			header?:  [string]: [...string]
 			trailer?: [string]: [...string]
+			statusCode: number
 		})
 `
 	testCases := map[string]struct {
```
Comment (FogDong) [inline pkg/workflow/providers/http/do_test.go:63]: Please add some test cases.

Comment (ArenaSu) [reply pkg/workflow/providers/http/do_test.go:63]: ok

**Review by wonderflow — APPROVED** [review]
good job

**Review by Somefive — APPROVED** [review]

**Pull request merged**